tests/unit/test_referee_folder.py

```diff
@@ -1,204 +1,253 @@
 from onitu.referee.folder import Folder
 
 
-def check_size(size, length):
-        folder = Folder('folder', (), None,
-                        file_size={'min': size, 'max': size})
-        assert folder.min_size == length
-        assert folder.max_size == length
+class FakeLogger(object):
```

Comment:
I didn't mean to include this in the PR. Forgot to fix it.
The problem is a Folder takes a logger and it is needed. How do we give it a Logger from within a unit test? Is there a best practice for this?